config/emojis.json

```diff
@@ -0,0 +1,338 @@
+[
+  {
+    "emoji": "🎨",
```

Comment:
I like the idea of emojis. Does this list need to live in this repo? 
Is there a way to easily extend the emoji list?

Reply:
Yes, It could be replaced with [an external link](https://github.com/carloscuesta/gitmoji-cli/blob/master/src/utils.js#L4)

Reply:
hey, it appears to be a breaking change.